Gems/EMotionFX/Code/EMotionFX/Pipeline/RCExt/Motion/MotionDataBuilder.cpp

```diff
@@ -375,9 +376,10 @@ namespace EMotionFX
 
                 // Set the bind pose transform.
                 SceneAPIMatrixType bindBoneTransformNoScale(bindSpaceLocalTransform);
-                const AZ::Vector3    bindPos   = coordSysConverter.ConvertVector3(bindSpaceLocalTransform.GetTranslation());
                 const AZ::Vector3    bindScale = coordSysConverter.ConvertScale(bindBoneTransformNoScale.ExtractScale());
-                const AZ::Quaternion bindRot   = coordSysConverter.ConvertQuaternion(AZ::Quaternion::CreateFromMatrix3x4(bindBoneTransformNoScale)).GetNormalized();
+                const AZ::Transform convertedbindTransform = AZ::Transform::CreateFromMatrix3x4(coordSysConverter.ConvertMatrix3x4(bindBoneTransformNoScale));
+                const AZ::Vector3    bindPos   = convertedbindTransform.GetTranslation();
+                const AZ::Quaternion bindRot = convertedbindTransform.GetRotation();
```

Comment:
nit: Prefer `bindPosition` and `bindRotation` while we're here, and same comment applies as above

In the test we could even just pass in the data that failed by pausing on a breakpoint in the real-world example and recording what the values were, then using that to pass to a free function in a test, that we then call here

Reply:
updated